src/algebraic_geometry/Scheme.lean

```diff
@@ -155,7 +155,7 @@ Spec.LocallyRingedSpace_map_comp f g
 /--
 The spectrum, as a contravariant functor from commutative rings to schemes.
 -/
-@[simps] def Spec : CommRingᵒᵖ ⥤ Scheme :=
+def Spec : CommRingᵒᵖ ⥤ Scheme :=
```

Comment:
```suggestion
-- We don't add lemmas via `@[simps]` as they are not useful.
def Spec : CommRingᵒᵖ ⥤ Scheme :=
```
Feel free to either hit merge on that suggestion, or ignore, or write your own, as any reviewer sees fit.

Reply:
(And if you do any of those, please update https://github.com/leanprover-community/mathlib4/pull/5040 if it hasn't already merged.)